docs/scripts/basic.md

```diff
@@ -1,32 +1,34 @@
 
 *Everyone shares their names, pronouns and purpose.*
 
 ## Process & Purpose
 
-Our main objective for the today is to come up with meaningful questions to ask the people whom we share our planet with. Some of these people are those who run for office and we are negotiating expectations with. Some are currently in office and we must help them uphold the will of the people. We shall ask questions to activists that increase our mutual understanding of each other. We may also need to help each other understand how different people come to the conclusions that they make and how to be free to make decisions that we can all be proud of.
+Our main objective for today is to invent meaningful questions to ask those we share our community with and those who represent us. We may ask those who run for office in the negotiation of expectations. We may ask those currently in office so we may help them uphold the will of the people. We may ask questions to activists that increase our mutual understanding of each other. We may also need to help each other understand how different people come to the conclusions reached. What is the least we need to do to be free to make decisions that benefit everyone in our community.
 
-We shall first plan together based on the complete interest of the group what we want to work on today.
-We shall then make a plan for how we want to work together today.
-Using our plan we shall practice investigating and researching different ways the issue can be approached.
-We shall make up questions that we could ask to specific people that might help us make places that we all want to live together in. 
+1. We shall first study the interest of the group.
 
-My purpose here as a facilitator is to make sure that we all get what matters to us as earthlings who want to understand each other and be represented by our elected officials. I am also here to make sure that we work towards our purpose and not against it. I may at times have to step in to remind members of the group the ground rules.
+2. Next we'll plan how to move forward based on the interest of the whole group.
+
+3. Using our plan we shall practice investigating and researching different ways the issue can be approached.
+
+4. We shall invent questions that we could ask to specific people that might help us create a world that we all want to live in and explore together.
+
+My purpose here as a facilitator is to make sure that we all get what matters to us as individuals who want to understand each other and be represented by our elected officials. More specifically I am also here to make sure that we synergistically achieve the objectives we agree to. I may at times have to remind members of the group the ground rules we all have agreed to abide.
```

Comment:
My purpose here as a facilitator is to make sure that we all get answers to what matters to us as individuals in mutual understanding , are represented by our elected officials, and that we synergistically achieve the objectives we agree to. I may at times have to remind members of the group the ground rules we all have agreed to abide.
 

 
Leave a comment